docs/en/operations/system-tables/users.md

```diff
@@ -0,0 +1,30 @@
+# system.users {#system_tables-users}
+
+User accounts parameters which is authorized in ClickHouse.
```

Comment:
- Parameters can't be authorized.
- Do we talk about authorized or about confirured users?

Maybe: "Contains a list of user accounts configured at the server."

Reply:
Yes, it's about that.